### ISSUE TITLE
requires bouncer dev-master

### ISSUE THREAD

**Issue by ibourgeois** [yadahan/nova-bouncer#5]
Should also allow: 

```
composer require silber/bouncer v1.0.0-rc.3
```

maybe (not tested): 

```
...
"require": {
        "php": "^7.1.3",
        "laravel/nova": "*",
        "silber/bouncer": "dev-master|v1.0.*"
},
...
```

**Comment by yakidahan** [comment]
The `v1.0.0-rc.3` of `silber/bouncer` does not include my [pull request](https://github.com/JosephSilber/bouncer/pull/356) that fixes the #3 issue.

**Comment by johan-steffens** [comment]
It seems like your [pull request](https://github.com/JosephSilber/bouncer/pull/356) was merged into master, so should be available on `v1.0.0-rc.4` of `silber/bouncer`

Can we please add
```
...
"require": {
        "php": "^7.1.3",
        "laravel/nova": "*",
        "silber/bouncer": "dev-master|v1.0.0-rc.4"
},
...
```

**Comment by yakidahan** [comment]
Hi @johan-steffens 

You're right! I updated the package and released a new version [v0.1.1](https://github.com/yadahan/nova-bouncer/releases/tag/v0.1.1)

**Comment by johan-steffens** [comment]
Awesome, thank you very much!

Appreciate the quick response 👍
